libc/src/errno/libc_errno.cpp

```diff
@@ -11,30 +11,44 @@
 
 namespace LIBC_NAMESPACE {
 
-#ifdef LIBC_TARGET_ARCH_IS_GPU
-struct ErrnoConsumer {
-  void operator=(int) {}
+struct Errno {
```

Comment:
This is already defined in `libc/src/errno/libc_errno.h`. Shouldn't we `#include "libc/src/errno/libc_errno.h"` instead of redefining the structure here?

Reply:
Done.